src/debuggers/natvis/magnum.natvis

```diff
@@ -38,16 +39,64 @@
     <DisplayString IncludeView="simple">{{ {_data[0],g}, {_data[1],g} }}</DisplayString>
     <DisplayString ExcludeView="simple">{{ x={_data[0],g} y={_data[1],g} }}</DisplayString>
   </Type>
+  <!-- Math::Vector2< Float > -->
+  <Type Name="Magnum::Math::Vector&lt;2,float&gt;">
+    <Intrinsic Name="Dot" Expression="_data[0]*_data[0]+_data[1]*_data[1]"></Intrinsic>
+    <Intrinsic Name="IsNormalized" Expression="Dot()&gt;0.99980000000500002&amp;Dot()&lt;1.0001999999950000"></Intrinsic>
+    <DisplayString IncludeView="simple">{{ {_data[0],g}, {_data[1],g} }}</DisplayString>
+    <DisplayString ExcludeView="simple" Condition="!IsNormalized()">{{ x={_data[0],g} y={_data[1],g} }}</DisplayString>
+    <DisplayString ExcludeView="simple" Condition="IsNormalized()">{{ UNIT VECTOR x={_data[0],g} y={_data[1],g} }}</DisplayString>
```

Comment:
For consistency with how the APIs are named I'd maybe say `NORMALIZED VECTOR` instead, or put a lowercase `(normalized)` in parentheses after. 

Being affter the value could make the actual numeric values aligned better, and that way people don't have to pay attention to the normalization property if they don't need to. I think.

Same in other cases.

Reply:
Makes sense. On it 👍

Reply:
I forgot to push the quaternions so there are separate commits for vectors and quaternions. Squash/fixup if desired.

Reply:
Another option for displaying this information is through an extra item in the expanded state. Requires you to open the item in the variables window, so not quite as compact, but it seems a bit more "idiomatic" (for whatever that's worth, even MS is rather inconsistent with their std natvis definitions).

Example here: https://github.com/mosra/corrade/blob/b192240cfeeb1d8a8a5d7fc13431480909cc9125/src/debuggers/natvis/corrade.natvis#L190

I think both options are fine, just throwing it out there.